pkg/monitor/ifindex.go

```diff
@@ -6,11 +6,23 @@ package monitor
 import (
 	"fmt"
 
-	"github.com/cilium/cilium/pkg/datapath/link"
+	"github.com/cilium/cilium/pkg/hubble/parser/getters"
 )
 
+// linkMonitor watches for links on the system so that ifindexes can be
+// string-formatted with human-readable interface names rather than the
+// integer values.
+//
+// Package-level variable just to avoid overhauling old code to introduce an
+// overarching structure to help format the messages nicely.
+var linkMonitor getters.LinkGetter
+
+func Init(linkGetter getters.LinkGetter) {
```

Comment:
I suppose there's no way around having a public Init method, but can we at least leave a comment describing where it's initialized from?